include/mlir/Dialect/AffineOps/AffineOps.td

```diff
@@ -212,9 +212,10 @@ def AffineIfOp : Affine_Op<"if", [ImplicitAffineTerminator]> {
   let regions = (region SizedRegion<1>:$thenRegion, AnyRegion:$elseRegion);
 
   let skipDefaultBuilders = 1;
+  
   let builders = [
     OpBuilder<"Builder *builder, OperationState *result, "
-              "Value *cond, bool withElseRegion">
+              "ArrayRef<Value *> args, IntegerSet set, bool withElseRegion">
```

Comment:
The order "IntegerSet set, ArrayRef<Value *> operands" will be more natural and consistent with other builders.